_includes/3-provisioning.md

```diff
@@ -185,13 +185,15 @@ This list is not exhaustive and is detailed in the <span class="h5">Request body
 The following HTTP request is sent from the provider to Ozwillo. 
 
 <pre>
-POST /apps/pending-instance/{instance_id} HTTP/1.1
+POST {instance_registration_path} HTTP/1.1
 Host: kernel.ozwillo-preprod.eu
 Accept: application/json, application/*+json
 Authorization: Basic {base64 encoding of client_id:client_secret}
 Content-Type: application/json;charset=UTF-8
 </pre>
 
+This request is indeed sent to the `instance_registration_uri` provided during [step #1](#s3-1-ozwillo-request) (and `instance_registration_path` is constructed from it by the HTTP client).
```

Comment:
Remove that paragraph, and replace the one before the request (line 185) with something like:

> The following HTTP request is sent from the provider to Ozwillo at the `instance_registration_uri` received in [step #1](#s3-1-ozwillo-request) (decomposed in `instance_registration_path` and `instance_registration_host` below).